test/integration/frame.cc

```diff
@@ -1298,3 +1298,65 @@ TEST(DOMFrame, WorldIncludeModel)
     EXPECT_EQ(expectedPoses[i], modelPose);
   }
 }
+
+////////////////////////////////////////
+// Test parsing a include element that has a pose element and includes a
+// submodel
+TEST(Frame, IncludeFrameWithSubmodel)
```

Comment:
As discussed in https://github.com/ignitionrobotics/sdformat/pull/596#pullrequestreview-688959663, this test will fail because the way flattening works has changed in `sdf9`. I think this test can be removed though since the functionality is tested elsewhere as part of frame semantics. @scpeters can you confirm?

Reply:
I'm not sure we have a test for multiple levels of nested model names. I'll see if I can update the test logic to use SemanticPose for the expectations

Reply:
updated to use `SemanticPose` API in https://github.com/ignitionrobotics/sdformat/pull/738/commits/d6fab5984a3b5ae0e8d6a551babad4b4a38c3c2c